src/styles/Colors.js

```diff
@@ -1,7 +1,7 @@
 export default Colors = {
-  primary: '#F4F4F4',
-  secondary: '#FFFFFF',
-  highlight: '#FF5553',
+  primary: '#89bdd3',
```

Comment:
yea that would be awesome to have like a legit graphic designer would probably spice it up a lot